viamonvif/discovery.go

```diff
@@ -109,6 +109,11 @@ type CameraInfo struct {
 	HardwareID      string   `json:"hardware_id"`
 }
 
+// Name creates generates a name for the camera based on discovered information about the camera.
+func (cam *CameraInfo) Name() string {
+	return fmt.Sprintf("%s_%s_%s", cam.Manufacturer, cam.Model, cam.SerialNumber)
+}
+
```

Comment:
no attachment on whether this gets used as the camera name for the generated cameras. open to suggestions if people have any preferences

Reply:
Makes sense to me. 

I think we are safe to assume that the name will be unique given the manufacturer and serial number are both included (pretty sure a given manufacturer guarantees unique serial numbers for their products).